onnxruntime/contrib_ops/rocm/bert/batched_gemm_softmax_gemm_permute_pipelines.cuh

```diff
@@ -355,19 +482,22 @@ class GemmSoftmaxGemmPermuteTunableOp : public tunable::TunableOp<GemmSoftmaxGem
  public:
   GemmSoftmaxGemmPermuteTunableOp();
 
-  inline static bool IsSupportedQkvFormat(const AttentionParameters* attn) {
-    switch (attn->qkv_format) {
-      case Q_K_V_BNSH:
-      case Q_K_V_BSNH:
-      case Q_KV_BSNH_BSN2H:
-      case QKV_BSN3H:
+  inline static bool IsSupportedMode(const RocmAttentionParameters* attn) {
+    switch (attn->mode) {
+      case QFMT_KFMT_VFMT_NONE_NONE_NONE_NONE:
+      case QFMT_KFMT_VFMT_2BNPH_NONE_2BNTH_NONE:
+        if (attn->qkv_format == Q_K_V_BNSH || attn->qkv_format == Q_K_V_BSNH) {
+          return true;
+        }
```

Comment:
> if (attn->qkv_format == Q_K_V_BNSH || attn->qkv_format == Q_K_V_BSNH) {
          return true;
        }

 Do we need a `break` or a `return false` here? Otherwise you'll fall through and `return true` anyway when the condition is not true.